tests/FormBuilderTest.php

```diff
@@ -189,6 +189,32 @@ public function testRenderRadioWithOldInput()
 		$this->assertEquals($expected, $result);
 	}
 
+	public function testRenderCheckboxWithOldInputAndForceUncheck()
```

Comment:
Can we rename these tests to something more like `testExplicitUncheckTakesPrecedenceOverOldInput`, and maybe add two more tests to prove they take precedence over model binding as well?